django/contrib/gis/geos/geometry.py

```diff
@@ -292,6 +292,14 @@ def contains(self, other):
         "Returns true if other.within(this) returns true."
         return capi.geos_contains(self.ptr, other.ptr)
 
+    def covers(self, other):
+        """
+        Returns true if the DE-9IM Intersection Matrix for the two geometries
```

Comment:
Most of Django could be updated, but I'm torn about whether or not we should do it (increased chance of backport conflicts / less useful git blame).